pkg/agent/manager/storage/storage.go

```diff
@@ -1,14 +1,20 @@
-package manager
+package storage
 
```

Comment:
This small refactor is necessary to avoid this import loop:

manager -> rotator -> node attestor -> manager

Reply:
i dont think we need this refactor anymore.

Reply:
I take that back, we still do for the `ServerStream` struct.